react-components/src/architecture/base/concreteCommands/SettingsCommand.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { BaseSettingsCommand } from '../commands/BaseSettingsCommand';
+import { SetQualityCommand } from './SetQualityCommand';
+import { SetPointSizeCommand } from './SetPointSizeCommand';
+import { SetPointColorTypeCommand } from './SetPointColorTypeCommand';
+import { SetPointShapeCommand } from './SetPointShapeCommand';
+import { PointCloudFilterCommand } from './PointCloudFilterCommand';
+import { type TranslateKey } from '../utilities/TranslateKey';
+
+export class SettingsCommand extends BaseSettingsCommand {
+  // ==================================================
+  // CONSTRUCTOR
+  // ==================================================
+
+  public constructor() {
+    super();
+
+    this.add(new SetQualityCommand());
+    this.add(new SetPointSizeCommand());
+    this.add(new SetPointColorTypeCommand());
+    this.add(new SetPointShapeCommand());
+    this.add(new PointCloudFilterCommand());
```

Comment:
If I understand correctly, this will add all point cloud operation (Point Size, Point color, point shape, etc) by default to the `Setting` button, assume if there is no point cloud model added into the `3D viewer`, as a user these option should populate only when point cloud model is in `viewer`.

Reply:
This is taken care of. There commands will not show up when no point cloud in the viewer.